components/external-link/global.css

```diff
@@ -1,19 +1,17 @@
-.external::after {
+.external {
   display: inline-flex;
+  column-gap: 0.25em;
```

Comment:
Making `.external` an `inline-flex` element affects all elements inside the link, not only the pseudo element, and this has changed the Specifications list to look like this:

<img width="427" alt="image" src="https://github.com/user-attachments/assets/63647ef5-e953-4416-836d-56fbef49157e" />

Example:

```html
<a class="external" href="https://html.spec.whatwg.org/multipage/semantics.html#attr-link-integrity" rel="noopener" target="_blank">HTML<br><small># attr-link-integrity</small></a>
```

Can we revert to `margin-left` on the pseudo element?